package.json

```diff
@@ -28,6 +28,14 @@
     "watchpack": "2.0.0-beta.13",
     "webpack-sources": "2.0.0-beta.8"
   },
+  "peerDependencies": {
+    "webpack-cli": "*"
+  },
```

Comment:
Remove this and keep only `peerDependenciesMeta`

Reply:
Done